resources/model/OpenStudio.idd

```diff
@@ -10971,10 +10971,19 @@ OS:AirTerminal:SingleDuct:VAV:HeatAndCool:NoReheat,
        \units m3/s
        \minimum> 0.0
        \autosizable
-  N2;  \field Zone Minimum Air Flow Fraction
+  N2,  \field Zone Minimum Air Flow Fraction
        \required-field
        \minimum 0.0
        \maximum 1.0
+  A6;  \field Minimum Air Flow Turndown Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note This field adjusts the design minimum flow rate by multiplying it using this schedule of fraction
+       \note values. This field is used with "Zone Minimum Air Flow Fraction". Schedule values are fractions
+       \note 0.0 to 1.0. This field adjusts the minimum airflow turndown below the design minimum air flow
+       \note and is intended for use with ASHRAE Standard 170. If this field is left blank, then the turndown
+       \note minimum air flow fraction value is set to 1 and the model uses the fixed fraction specified in
+       \note in the field "Zone Minimum Air Flow Fraction".
```

Comment:
Have you considered making this a required-field and setting to alwaysOnDiscreteSchedule instead? (this comment applies to all of the objects)

Reply:
For consistency, I think we should follow the other schedule fields (i.e., not making it a required-field and setting to alwaysOnDiscreteSchedule).